notifications/query.go

```diff
@@ -0,0 +1,80 @@
+package notifications
+
+import (
+	"encoding/json"
+
+	context "github.com/jbenet/go-ipfs/Godeps/_workspace/src/code.google.com/p/go.net/context"
+	peer "github.com/jbenet/go-ipfs/p2p/peer"
+)
+
+const RoutingQueryKey = "RoutingQueryEvent"
```

Comment:
My original thought was that we could eventually have just a `RegisterForNotifications` that took a key, so i made it public.. probably doesnt need to be.